Source/Flow/Public/Interfaces/FlowPredicateInterface.h

```diff
@@ -0,0 +1,30 @@
+// Copyright https://github.com/MothCocoon/FlowGraph/graphs/contributors
+
+#pragma once
+
+#include "UObject/Interface.h"
+#include "Templates/SubclassOf.h"
+
+#include "FlowPredicateInterface.generated.h"
+
+class UFlowNodeAddOn;
+
+// Predicate interface for AddOns
+UINTERFACE(MinimalAPI, BlueprintType, Blueprintable, DisplayName = "Flow Predicate Interface")
+class UFlowPredicateInterface : public UInterface
+{
+	GENERATED_BODY()
+};
+
+class FLOW_API IFlowPredicateInterface
```

Comment:
This is an interface for Flow "Predicate" AddOns that can be used by the "Branch" flow node.